package.json

```diff
@@ -11,12 +11,16 @@
     "Other"
   ],
   "activationEvents": [
+    "onView:FileTreeView",
     "onCommand:onevscode.build",
     "onCommand:onevscode.import",
     "onCommand:onevscode.json-tracer",
     "onCommand:onevscode.configuration-settings",
     "onCommand:onevscode.toggle-codelens",
     "onCommand:onevscode.circle-tracer",
+    "onCommand:onevscode.fileTreeView-open",
```

Comment:
How about adding this changes when you implement a real operation?

Reply:
Removing the line seems not good since we have a button that triggers the event.
Instead, I added empty event handlers. PTAL.